public/assets/splash-screen.js

```diff
@@ -1,11 +1,11 @@
 var isDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
 var splashScreen = document.getElementById('splash-screen');
 
-// set accent.background-secondary
+// set ink.background-secondary
 if (isDark) {
-  // darkModeInk.2
+  // oldDarkModeInk.2
   splashScreen.style.backgroundColor = '#2C2A24';
 } else {
-  // lightModeInk.3
+  // oldLightModeInk.3
```

Comment:
I asked Anas to add comments here to link these colours to tokens as we couldn't import tokens originally when he was doing this feature.

Now he has changed his code to use JS maybe we can instead import these? We could change this comment to be as below

```suggestion
  // text-primary_dark
```

I couldn't find a semantic colour for `#2C2A24` so we could need to replace that. 

It's not a huge deal but that would add the last piece to tokenised semantic colours. 

I asked Anas here: https://github.com/leather-wallet/extension/pull/4935/files#r1498680541

Reply:
I can see he hardcoded what were the values for the old `background-secondary`. I will update these with the new values and fix the comments, thx!

Reply:
Thanks!